extension/Readme.md

```diff
@@ -0,0 +1,16 @@
+# Timescale-Prometheus Support Extension #
+
+This [Postgres extension](https://www.postgresql.org/docs/12/extend-extensions.html)
+contains support functions to improve the performance of Timescale-Prometheus.
+While Timescale-Prometheus will run without it, adding this extension will
+cause it perform better.
+
+## Installation ##
+
+The extension is installed by default on the timescale-prometheus docker image,
+and will be installed automatically by timescale-prometheus, if it has
```

Comment:
I wouldn't put the ```CREATE EXTENSION timescale_prometheus_support;``` instruction here because the connector and not the use should call that. I'd just put the make && make install here and say that the prometheus-connector will call CREATE EXTENSION

Reply:
oh, whoops, I thought I'd deleted that line 😅